meinberlin/react/contrib/Card.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import django from 'django'
+
+import { CardStats } from './CardStats'
+
+const translated = {
+  detailsStr: django.gettext('Item details'),
+  toStr: django.gettext('to:')
+}
+
+export const Card = (props) => {
+  const { item, permissions, children } = props
+
+  return (
+    <article className="modul-card" data-add-clickable-area="smart">
+      <header className="card__header">
+        <h3 className="title">{item.name}</h3>
+        {children}
+      </header>
+      <section className="card__footer--light">
```

Comment:
not sure if this should be a footer but not sure if that makes sense without a body/section, however we might still need a body depending if there's additional content for the other list cards so this could just be something to keep in mind